crates/ra_mbe/src/mbe_expander.rs

```diff
@@ -221,11 +221,13 @@ fn match_lhs(pattern: &crate::Subtree, input: &mut TtCursor) -> Result<Bindings,
                 }
                 _ => return Err(ExpandError::UnexpectedToken),
             },
-            crate::TokenTree::Repeat(crate::Repeat { subtree, kind: _, separator }) => {
+            crate::TokenTree::Repeat(crate::Repeat { subtree, kind, separator }) => {
```

Comment:
This bit of code should probably just be rewritten to handle repetition properly.

I haven't managed to understand how rustc determines when it should stop expanding repeats, so that's why I written this random code that handles at least some cases.